src/main/java/com/sourcegraph/javagraph/ClassPathUtil.java

```diff
@@ -0,0 +1,63 @@
+package com.sourcegraph.javagraph;
+
+import com.google.common.io.Files;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+public class ClassPathUtil {
```

Comment:
removed this file in alexsaveliev/srclib-java@0b25677b36904e61ee0dd0eb1a4796d45f83613a